### PR TITLE
Made required attribute work

### PR DESCRIPTION
**Pull request by bolelamx**
Issue #83

### CODE REVIEW

File: src/css/ng-wig.css [hunk 1]
```diff
@@ -77,30 +77,25 @@
  *
  */
 .nw-editor {
-  display: block;
-  position: relative;
-  overflow-y: auto;
-  padding: 0 8px;
+  display: table;
```
Comment (stevermeister) [inline src/css/ng-wig.css]: hm, not clear, could you put some explanations here?

File: src/javascript/app/ng-wig/ngWigDirective.js [hunk 2]
```diff
@@ -3,15 +3,16 @@ angular.module('ngWig')
 
     return {
       scope: {
-        content: '=ngWig'
+        content: '=ngWig',
+        name: '@'
```
Comment (stevermeister) [inline src/javascript/app/ng-wig/ngWigDirective.js]: it's not mandatory field, right?

**Comment by stevermeister** [comment]
does not this solution (http://jsfiddle.net/Tentonaxe/V4axn/) work for you case?

File: src/css/ng-wig.css (hunk 1 above)
Comment (besuhoff) [inline src/css/ng-wig.css]: this is a fix for auto-expand feature that we've suddenly lost in one of the previous releases

File: src/javascript/app/ng-wig/ngWigDirective.js [hunk 3]
```diff
@@ -9,9 +9,10 @@ angular.module('ngWig')
       replace: true,
       templateUrl: 'ng-wig/views/ng-wig.html',
       link: function (scope, element, attrs) {
-
+        scope.name = attrs.name;
```
Comment (stevermeister) [inline src/javascript/app/ng-wig/ngWigDirective.js]: it's good to have better property name
like formElementName

**Comment by besuhoff** [comment]
:+1:

**Comment by stevermeister** [comment]
:+1:

**Pull request merged**